### PR TITLE
Maintenance: Upgrade versions + remove asset classification links

### PR DESCRIPTION
**Pull request by hyperschwartz**
# Description
This PR moves the versions that this application uses to the latest values, notably upgrading Provenance Protos from a RC version to the realness.  Importantly, this PR also removes Asset Classification Smart Contract backwards compatibility.  The AC contract will still be able to talk to the proper gateway, but now all events have to come through the OS gateway events (which all current versions of the AC contract use).

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,95 +1,98 @@
 [versions]
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
```
Comment (hyperschwartz) [inline gradle/libs.versions.toml:2]: Some version bumps, but mostly reorganization to the dank hyphenated format that makes things dot-qualified in the build files.

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,95 +1,98 @@
...
+bouncycastle = "1.70"
 exposed = "0.37.3"
+figure-eventstream = "0.7.1"
```
Comment (hyperschwartz) [inline gradle/libs.versions.toml:5]: Version bump to use new ES libs from figure tech

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -1,95 +1,98 @@
...
-grpcNetty = { module = "io.grpc:grpc-netty-shaded", version.ref = "grpc" }
-kotlinModule = { module = "com.fasterxml.jackson.module:jackson-module-kotlin", version.ref = "jackson" }
-protobufModule = { module = "com.hubspot.jackson:jackson-datatype-protobuf", version.ref = "jacksonProtobuf" }
-allOpen = { module = "org.jetbrains.kotlin:kotlin-allopen", version.ref = "kotlin" }
-coroutinesCoreJvm = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm", version.ref = "kotlinCoroutines" }
-coroutinesReactor = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-reactor", version.ref = "kotlinCoroutines" }
-coroutinesJdk8 = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-jdk8", version.ref = "kotlinCoroutines" }
-coroutinesSLF4J = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-slf4j", version.ref = "kotlinCoroutines" }
+# Core
+asset-model = { module = "io.provenance.model:metadata-asset-model", version.ref = "asset-model" }
```
Comment (hyperschwartz) [inline gradle/libs.versions.toml:30]: Moved this from a direct ref in a build file to this file, just for consolidation's sake

File: gradle/libs.versions.toml [hunk 4]
```diff
@@ -1,95 +1,98 @@
...
-protobuf = ["protobufJava", "protobufJavaUtil"]
-provenance = ["provenanceEventStreamApi", "provenanceEventStreamApiModel", "provenanceEventStreamCli", "provenanceEventStreamCore", "provenanceGrpcClient", "provenanceHdWallet", "provenanceProto", "scopeSdk", "scopeUtil"]
-scarlet = ["messageAdapterMoshi", "scarlet", "streamAdapterCoroutines", "webSocketOkHttp"]
-springboot = ["springBootAutoConfigure", "springBootStarter", "springBootStarterActuator", "springBootStarterAOP", "springBootStarterJetty", "springBootStarterWeb"]
-exposed = ["exposedCore", "exposedDao", "exposedJdbc", "postgres", "hikariCP", "sqLite", "flyway"]
-testKotlin = ["kotlinTest", "mockk"]
-testSpringBoot = ["starterTest"]
-jwt = ["javaJwt"]
-testContainers = ["testContainers", "testContainersPostgres"]
+database = ["exposed-core", "exposed-dao", "exposed-jdbc", "flyway", "hikari", "postgres", "sqlite"]
```
Comment (hyperschwartz) [inline gradle/libs.versions.toml:87]: Renamed from `exposed` to be more accurate about what this brings in

**Review by piercetrey-figure — APPROVED** [review]

**Pull request merged**